Dockerfile

```diff
@@ -3,7 +3,9 @@ FROM caddy:2.10.0-builder-alpine AS builder
 ENV CADDY_HASH=d82618e1407aacb3a3fbc1a8c0ee58e895337c1b
 
 RUN set -ex; \
-    xcaddy build --with github.com/porech/caddy-maxmind-geolocation@"$CADDY_HASH"
+    xcaddy build --with github.com/porech/caddy-maxmind-geolocation@"$CADDY_HASH" \
+        --with github.com/mholt/caddy-l4
```

Comment:
needs to pin the version, follow up to https://github.com/szaimen/aio-caddy/pull/76